docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -9,104 +9,57 @@ You need a Kubernetes or Openshift cluster, and the `kubectl` or `oc` command-li
-----
-oc apply -f https://repository.hazelcast.com/operator/bundle-{full-version}.yaml
-----
---
-====
-
-TIP: If you always want to use the latest release, you can use the `bundle-latest.yaml` file.
-
-At this point, the Hazelcast Platform Operator should be up and running. You can check it with the command below.
+. Add the link:https://github.com/hazelcast/charts[Hazelcast Helm Charts repository] to your Helm repository list by running the following command:
 
-[tabs]
-====
-Kubernetes::
 +
---
 [source,shell]
 ----
-kubectl logs deployment.apps/hazelcast-platform-controller-manager
+helm repo add hazelcast https://hazelcast-charts.s3.amazonaws.com/
+helm repo update
 ----
---
-Openshift::
-+
---
-[source,shell]
-----
-oc logs deployment.apps/hazelcast-platform-controller-manager
-----
---
-====
 
-```
-2021-07-14T09:03:40.713Z        INFO    setup   Watching namespace: default
-2021-07-14T09:03:41.524Z        INFO    controller-runtime.metrics      metrics server is starting to listen    {"addr": ":8080"}
-2021-07-14T09:03:41.524Z        INFO    setup   starting manager
-2021-07-14T09:03:41.525Z        INFO    controller-runtime.manager      starting metrics server {"path": "/metrics"}
-```
+. You can either deploy the Hazelcast Platform Operator at the same time as CRDs or separately.
 
-== Optional: Customize the Namespace
++
+NOTE: Since CRDs are global resources, they may need to be installed by an administrator.
 
-By default, Hazelcast Operator is installed into the `default` namespace. If you want to change it, you can either replace all `namespace: default` parameters in `bundle.yaml` or use https://kustomize.io/[Kustomize] to achieve it by creating the following `kustomization.yaml` file.
+.. Run the following command to *deploy the operator and the CRDs together*
 
-[source,yaml]
++
+[source,shell]
 ----
-namespace: <your-namespace>
-resources:
-- bundle.yaml
+helm install operator hazelcast/hazelcast-platform-operator --set installCRDs=true
 ----
 
-Then, use Kustomize to apply `bundle.yaml` with the changed namespace.
+.. Run the following commands to *deploy the operator and the CRDs separately*
 
-[tabs]
-====
-Kubernetes::
 +
---
 [source,shell]
 ----
-kubectl apply -k .
+helm install operator-crds hazelcast/hazelcast-platform-operator-crds
 ----
---
-Openshift::
+
++
+After installing CRDs, the operator can be installed by running the following command:
+
 +
---
 [source,shell]
 ----
-oc apply -k .
+helm install operator hazelcast/hazelcast-platform-operator
 ----
---
-====
-
-If you deploy multiple Hazelcast Operators in different namespaces, you need a way to differentiate the name of each operator. For example, you can give each operator's name a suffix by adding the `nameSuffix` field to the `kustomization.yaml` file:
 
-[source,yaml]
++
+. Monitor the operator logs
++
+At this point, the Hazelcast Platform Operator should be up and running. You can check it with the command below.
++
+[source,shell]
```

Comment:
I think we should add an Openshift tab here, as we discussed.

Reply:
Sure, I forgot to re-add it.